### PR TITLE
Add routes 

### LINKED ISSUE #29: Routing
## Set up routing between pages

- [ ] fix routing in App.js between splash & login

### PR DESCRIPTION
**Pull request by Phil9219**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
-// import ChooseYourRunPage from "../src/components/Page/ChooseYourRunPage";
-// import CreatedRunPage from "./components/Page/CreatedRunPage";
-// import RunsDonePage from "./components/Page/RunsDonePage";
+import SplashPage from "../src/components/Page/SplashPage";
+import LoginPage from "../src/components/Page/LoginPage";
+import ChooseYourRunPage from "../src/components/Page/ChooseYourRunPage";
+import CreatedRunPage from "./components/Page/CreatedRunPage";
+import RunsDonePage from "./components/Page/RunsDonePage";
 import RunSetupPage from "./components/Page/RunSetupPage";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
```
Comment (boldCM) [inline client/src/App.js:9]: it seems like you don't need to go up to the src level for the import ;)

File: client/src/components/Page/RunSetupPage.js [hunk 2]
```diff
@@ -45,7 +47,9 @@ export default function RunSetupPage() {
           <Input placeholder="Choose Your Distance" />
           <Input placeholder="Name Your Run" />
         </InputfieldsContainer>
-        <Button>Create Run</Button>
+        <Link to="/created run" label="create">
```
Comment (boldCM) [inline client/src/components/Page/RunSetupPage.js:50]: what's the label for in here?

**Review by boldCM — APPROVED** [review]
lgtm 🎉

**Review by ncbecker — APPROVED** [review]
lgtm 👍  I would maybe name the path with underscores instead of blanks

File: client/src/App.js (hunk 1 above)
Comment (Phil9219) [reply client/src/App.js:9]: Thanks :)

**Comment by Phil9219** [comment]
> lgtm 👍 I would maybe name the path with underscores instead of blanks

**Comment by Phil9219** [comment]
Thanks. Will use underscores. :)

**Pull request closed without merging**